File: Python/A_NivelIniciante/Exercicio_1038.py
linha1 = input()
cod,qtd = linha1.split(" ")
cod = int(cod)
qtd = int(qtd)
total = 0
if cod==1:
    total = qtd * 4.00
elif cod==2:
    total = qtd * 4.50
elif cod==3:
    total = qtd * 5.00
elif cod==4:
    total = qtd * 2.00
elif cod==5:
    total = qtd * 1.50  
print("Total: R$ %.2f" %total)

File: Javascript/A_NivelIniciante/Exercicio_1038.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var linha1 = lines[0].split(" ");
var cod_produto = parseFloat(linha1[0])
var qtd = parseFloat(linha1[1]);
var precos = [4.00,4.50,5.00,2.00,1.50];
console.log("Total: R$ " + (precos[cod_produto-1] * qtd).toFixed(2));